.travis.yml

```diff
@@ -29,4 +29,5 @@ script:
   - doxygen doxygen/doxygen.conf
   - grep -v "$IGNORE" doxygen-build.txt | tee tmp.txt
   - test ! -s tmp.txt
+  - .travis/push-docs.sh
```

Comment:
As far as i understand, this is only generating the doxygen related things, right? I think the name `push-docs.sh` is misleading then, as it doesn't touch our [our neomutt guide](https://neomutt.org/guide/index.html).